packages/storybook/.storybook/styles.css

```diff
@@ -23,3 +23,229 @@ html {
     background-color: black;
   }
 }
+
+:root {
```

Comment:
I understand this is temporary? And ideally, we should want to import the theme from the library instead of this code?